inference-engine/src/low_precision_transformations/src/fake_quantize.cpp

```diff
@@ -124,42 +127,78 @@ std::shared_ptr<opset1::FakeQuantize> FakeQuantizeTransformation::fuseElementwis
+                fakeQuantize->get_input_node_shared_ptr(2),
+                fakeQuantize->get_input_node_shared_ptr(3),
+                fakeQuantize->get_input_node_shared_ptr(4),
+                fakeQuantize->get_levels() ));
+            auto zeroMappingValues = as_type_ptr<opset1::Constant>(zeroMapping)->cast_vector<float>();
+
+            const bool inputIntervalsConstBroadcasted = inputIntervalsConstRank < resultRank;
+            if (inputIntervalsConstBroadcasted) {
+                inputLowConstValues = std::vector<float>(resultRank, inputLowConstValues[0]);
+                inputHighConstValues = std::vector<float>(resultRank, inputHighConstValues[0]);
+            }
+            const bool outputIntervalsConstBroadcasted = outputIntervalsConstRank < resultRank;
+            if (outputIntervalsConstBroadcasted) {
+                outputLowConstValues = std::vector<float>(resultRank, outputLowConstValues[0]);
+                outputHighConstValues = std::vector<float>(resultRank, outputHighConstValues[0]);
+            }
+            const bool multiplyConstBroadcasted = multiplyConstRank < resultRank;
+            for (size_t i = 0; i < resultRank; ++i) {
+                const float denominator = valueVec[multiplyConstBroadcasted ? 0ul : i];
+                if ((denominator == 0.f) || (std::abs(denominator) < 1.e-32)) {
+                    outputLowConstValues[i] = zeroMappingValues[i];
+                    outputHighConstValues[i] = zeroMappingValues[i];
+                } else {
+                    inputLowConstValues[i] /= denominator;
+                    inputHighConstValues[i] /= denominator;
+                }
+            }
+
+            const Shape resultShape = multiplyConstBroadcasted ? inputLowConst_f32->get_output_shape(0) : constant->get_output_shape(0);
+            inputLowConst_f32 = std::make_shared<opset1::Constant>(element::f32, resultShape, inputLowConstValues);
+            inputHighConst_f32 = std::make_shared<opset1::Constant>(element::f32, resultShape, inputHighConstValues);
+            outputLowConst_f32 = std::make_shared<opset1::Constant>(element::f32, resultShape, outputLowConstValues);
+            outputHighConst_f32 = std::make_shared<opset1::Constant>(element::f32, resultShape, outputHighConstValues);
+        } else {
+            inputLowConst_f32 = fold<opset1::Divide>(inputLowConst_f32, value);
+            inputHighConst_f32 = fold<opset1::Divide>(inputHighConst_f32, value);
         }
 
-        inputLowConst_f32 = fq::updateShape(fold<opset1::Divide>(inputLowConst_f32, value), fakeQuantize->get_output_shape(0));
-        inputHightConst_f32 = fq::updateShape(fold<opset1::Divide>(inputHightConst_f32, value), fakeQuantize->get_output_shape(0));
+        inputLowConst_f32 = fq::updateShape(inputLowConst_f32, fakeQuantize->get_output_shape(0));
+        inputHighConst_f32 =  fq::updateShape(inputHighConst_f32, fakeQuantize->get_output_shape(0));
```

Comment:
Looks like redundant space
```suggestion
        inputHighConst_f32 = fq::updateShape(inputHighConst_f32, fakeQuantize->get_output_shape(0));
```